inline-java.cabal

```diff
@@ -68,3 +68,4 @@ test-suite spec
     inline-java,
     text
   default-language: Haskell2010
+  ghc-options: -fplugin=Language.Java.Inline.Plugin -DHSPEC_DISCOVER=hspec-discover
```

Comment:
`inline-java` is supposed to indicate `-fplugin=Language.Java.Inline.Plugin` via TH. Is this not working?

Reply:
nah, I'm dreaming. We still have to add the calls to `Language.Haskell.TH.Syntax.addCorePlugin`.